be/src/olap/column_predicate.h

```diff
@@ -262,6 +269,8 @@ class ColumnPredicate {
     // TODO: the value is only in delete condition, better be template value
     bool _opposite;
     std::shared_ptr<PredicateParams> _predicate_params;
+    mutable uint64_t _evaluated_rows = 1;
```

Comment:
The column predicate will be used by multi scanner thread.  So it is immutable in the past. If you add some state(_evaluated_rows, _passwd_rows)， it may be not right or may core?

Reply:
Currently, it is better to collect this info in segment_iterator during evaluate stage.

Reply:
This part was previously inside the bloomfilter, and now it has been mentioned on the base class. There shouldn't be any significant issues.